Add additional info in introduction chapter (#66)
infinum/ngx-nuts-and-bolts: ngx-nuts-and-bolts-docs/docs/introduction.md

slug: /
---

This NPM library contains reusable components and helpers to ease and speed up development and testing of angular applications.

## 1. Installation and setup

```ts
npm install @infinumjs/ngx-nuts-and-bolts
```

> After the library is installed you can simply import already implemented functionalities in the files you wish to use them in.

Reviewer: ```suggestion
After the library is installed you can simply import what you need.
```

> You can browse through the documentation to see the examples of usage of implemented nuts-and-bolts.

Reviewer: ```suggestion
You can browse through the documentation to see which nuts-and-bolts are available and how to use them.
```
